### PR TITLE
Parameterize backup restore script

### PR DESCRIPTION
**Pull request by hubertdeng123**
Allows users to call
```
./scripts/backup.sh (users/organizations/config)
./scripts/restore.sh (users/organizations/config)
```

### CODE REVIEW

File: scripts/_lib.sh [hunk 1]
```diff
@@ -60,13 +60,15 @@ function reset() {
 }
 
 function backup() {
+  type=$1
```
Comment (aminvakil) [inline scripts/_lib.sh:63]:
```suggestion
  type=${1:-"global"}
```

**Review by aminvakil — COMMENTED** [review]
To keep it backward compatible for anyone who might use this script some another way.

**Comment by ricardoapaes** [comment]
It will be of great help to us. I'm waiting for the merge!

File: scripts/_lib.sh (hunk 1 above)
Comment (hubertdeng123) [reply scripts/_lib.sh:63]: good suggestion!

**Review by azaslavsky — APPROVED** [review]
We should update the docs to mention this as well

**Comment by hubertdeng123** [comment]
> We should update the docs to mention this as well
agreed

**Pull request merged**